docs/RATE_LIMITS.md

```
@@ -0,0 +1,164 @@
+| Field | Description |
+|-------|-------------|
+| `maxAmount` | Maximum allowed amount at any time |
+| `slope` | Rate at which the limit increases [tokens / second] |
+| `lastAmount` | Amount left available at the last update |
+| `lastUpdated` | Timestamp when the rate limit was last updated |
+
+## Rate Limit Calculation
+
+The current rate limit is calculated as:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)